_posts/2017-06-07-hello-world.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Hello World!"
+date:   2017-06-07 20:32:13 +0000
+---
+
+Welcome to our brand new ACSS website!
+
+As well as the site, we've made a few changes to the way the society works and we're hoping they'll really improve your experience. Let's take a look...
+
+## Code of Conduct
+TODO
+
+## Mailing List
+We know we've not been the best at communication in the past, and we want to change that. Starting in the new academic year, we'll be using our new mailing list to let you know about all our upcoming events, as well as other regular updates. You can sign up for our mailing list [here][Mail].
+
+## Slack
+The committee has been using Slack internally for some while now, both to help organise our events and to socialise. This year we've decided to open it up to everybody, and you can find it [here][Slack]. Signups are open for all `@aston.ac.uk` emails.
+
+We hope this will act as an informal way of holding discussions, and compliment our Facebook group as a less structured alternative. Please be mindful of our Code of Conduct.
+
+## Social Media
+We've also had a shake up of our social media accounts. All events and announcements will now be posted first and foremost on our [Facebook page][Facebook], so make sure to follow it. We'll also be posting these on our [Twitter][Twitter] account, and in our members only [Facebook group][Facebook Group].
+
+We'll try our best to maintain consistent content between the different platforms, but sometimes things like retweets don't make it beyond Twitter!
+
+## Calendar
+In the past, we've maintained a calendar of events. Starting this year, all events will have a Facebook event, allowing us to generate our calendar from Facebook. For those without accounts on Facebook, this can be accessed as [a page][Calendar] on our site.
+
+## Finally
+We hope these changes will make being a part of the society even better! If you have any questions or concerns, please reach out to us via your preferred contact method.
+
+We're hard at work planning a programme of amazing events for next term, and we can't wait to share some of it with you. Until then, have a great summer!
+
+[Mail]: http://twitter.us15.list-manage.com/subscribe?u=5c76c6487162a4644f24574b6&id=8948e9a3d3
+[Twitter]: https://jekyllrb.com/docs/home
```

Comment:
[Twitter]: https://twitter.com/AstonCSS
[CoC]: https://github.com/aston-css/code-of-conduct